packages/metals-vscode/package.json

```diff
@@ -651,19 +653,29 @@
         "title": "Stop Scala CLI BSP server"
       },
       {
-        "command": "metals.toggle-implicit-conversions-and-classes",
+        "command": "metals.toggle-implicit-conversions",
         "category": "Metals",
-        "title": "Toggle showing implicit conversions and classes"
+        "title": "Toggle inlay hints for implicit conversions"
       },
       {
         "command": "metals.toggle-implicit-parameters",
         "category": "Metals",
-        "title": "Toggle showing implicit parameters"
+        "title": "Toggle inlay hints for implicit parameters"
+      },
+      {
+        "command": "metals.toggle-inferred-types",
+        "category": "Metals",
+        "title": "Toggle inlay hints for inferred type"
       },
       {
-        "command": "metals.toggle-show-inferred-type",
+        "command": "metals.toggle-type-parameters",
```

Comment:
Could we try to avoid rename the methods, it might break for people if they have shortcuts set up.